pkg/strategy/bollmaker/strategy.go

```diff
@@ -653,6 +671,36 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 			return
 		}
 
+		// Update spreads
+		if s.DynamicSpreadWindow > 0 && kline.Direction() == types.DirectionUp {
+			s.DynamicAskSpread.Update(kline.GetHigh().Sub(kline.GetOpen()).Div(kline.GetOpen()).Float64())
```

Comment:
maybe create variables in float64 first, and then do the calculation?  
ex: 
```go
high := kline.High.Float64()
open := kline.Open.Float64()
```

Reply:
so that in line 682 we don't need to convert the float64 back to fixedpoint and do slow comparison.